### PR TITLE
Fix build problems

### PR DESCRIPTION
**Pull request by gnodet**
Maven 4 is a bit more restrictive, so the build should be cleaned a bit

### CODE REVIEW

File: components/camel-aws/camel-aws2-cw/pom.xml [hunk 1]
```diff
@@ -80,9 +80,5 @@
             <type>test-jar</type>
             <scope>test</scope>
         </dependency>
-        <dependency>
-            <groupId>org.apache.camel</groupId>
-            <artifactId>camel-health</artifactId>
-        </dependency>
```
Comment (oscerd) [inline components/camel-aws/camel-aws2-cw/pom.xml:86]: This is needed for health check to work

Comment (gnodet) [reply components/camel-aws/camel-aws2-cw/pom.xml:86]: It's a dup. It's already defined above in the pom.

Comment (oscerd) [reply components/camel-aws/camel-aws2-cw/pom.xml:86]: Well spotted. I shouldn't do review from the bed

**Review by oscerd — APPROVED** [review]

**Review by davsclaus — APPROVED** [review]

**Comment by davsclaus** [comment]
is there anything to fix as well in camel-spring-boot or camel-karaf ?

**Comment by gnodet** [comment]
> is there anything to fix as well in camel-spring-boot or camel-karaf ?

I'll try to have a look.

**Pull request merged**